Core/src/org/sleuthkit/autopsy/coreutils/ImageUtils.java

```diff
@@ -726,8 +753,20 @@ private void saveThumbnail(BufferedImage thumbnail) {
                 if (image.isError() == false) {
                     return image;
                 }
-                //fall through to default image reading code if there was an error
+            } else if (file.getNameExtension().toLowerCase().compareTo("tec") == 0) {
+                int offset = 0;
+                if (isJfifFileHeaderWithLeadingEOIMarker(file)) {
```

Comment:
I think using the Image constructor directly means we are missing out of the progress indicator.  I wonder if there is a way redesign things for gif and tec so that we still get it?